server/src/main/java/org/elasticsearch/action/admin/cluster/repositories/cleanup/CleanupRepositoryRequest.java

```diff
@@ -20,18 +22,34 @@ public class CleanupRepositoryRequest extends AcknowledgedRequest<CleanupReposit
 
     private String repository;
 
-    public CleanupRepositoryRequest(String repository) {
-        super(TRAPPY_IMPLICIT_DEFAULT_MASTER_NODE_TIMEOUT, DEFAULT_ACK_TIMEOUT);
+    public CleanupRepositoryRequest(TimeValue masterNodeTimeout, TimeValue ackTimeout, String repository) {
+        super(masterNodeTimeout, ackTimeout);
         this.repository = repository;
     }
 
-    public CleanupRepositoryRequest(StreamInput in) throws IOException {
-        super(TRAPPY_IMPLICIT_DEFAULT_MASTER_NODE_TIMEOUT, DEFAULT_ACK_TIMEOUT);
+    public static CleanupRepositoryRequest readFrom(StreamInput in) throws IOException {
```

Comment:
This request class was missing the `AcknowledgedRequest` header on the wire.

Reply:
No need I think, we'll be clearing up the transport versions in v9 anyway and that'll expose cases like this.